composer.json

```diff
@@ -30,7 +30,7 @@
         "guzzlehttp/psr7": "^1.0",
         "imagine/imagine": "^0.6 || ^0.7",
         "jms/serializer-bundle": "^1.0 || ^2.0",
-        "knplabs/gaufrette": "^0.3 || ^0.4 || ^0.5 || ^0.6 || ^0.7",
+        "knplabs/gaufrette": "^0.3 || ^0.4 || ^0.5 || ^0.6 || ^0.7 || ^0.8",
```

Comment:
Sigh… when are they going to release version 1? Before the 120 chars limit I hope!

Reply:
Lol I already complained last time: #1477